pallets/aggregate/src/should.rs

```diff
@@ -498,11 +498,13 @@ mod aggregate {
                 aggregation_id,
                 aggregation,
                 destination,
+                owner_account,
                 ..
             } = MockDispatchAggregation::pop().expect("No call received");
 
             assert_new_receipt(domain_id, aggregation_id, Some(aggregation));
             assert_eq!(hyperbridge_destination(), destination);
+            assert_eq!(USER_DELIVERY_OWNER, owner_account);
```

Comment:
Where are we checking the delivery owner balance after the dispatch ?

Reply:
adding it here